tests/integration/conftest.py

```diff
@@ -210,17 +209,6 @@ def credentials_mapping_fixture():
                 password="wrong-password",
             ),
         ),
-        "sqlite_datasource": CredentialModel(
-            name="sqlite_datasource",
-            feature_store_id=ObjectId(),
-        ),
-        "databricks_featurestore": CredentialModel(
-            name="databricks_featurestore",
-            feature_store_id=ObjectId(),
-            database_credential=AccessTokenCredential(
-                access_token=os.getenv("DATABRICKS_ACCESS_TOKEN", ""),
-            ),
-        ),
```

Comment:
The integration test suite does still support databricks source type: https://github.com/featurebyte/featurebyte/blob/54e5f035a84b5021d849b3220a9f06503a71ba81/tests/source_types.py#L7

Just that we don't run it by default. Is there any downside of keeping this so that we can still test it if needed? No concern on removing sqlite though.